### PR TITLE
#184581348 buyer adds product reviews

### PR DESCRIPTION
**Pull request by INValentin**
#### What does this PR do?
This PR is to allow buyers to review products by providing a rating and feedback. They also can get, update, or delete their reviews.
#### Description of Task to be completed?
A buyer should be able to provide a review on a product they have bought successfully, rating  and feedback.
#### How should this be manually tested?

- run `npm install && npm run migrate:all && npm run start`
- Then access the API docs on `/api-docs`, from there you can `test the api`.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker/Trello stories?
https://www.pivotaltracker.com/n/projects/2632830/stories/184581348
#### Screenshots (if appropriate)
Screenshot 2023-03-20 at 20 25 20
Screenshot 2023-03-20 at 20 27 03

#### Questions:
N/A

### CODE REVIEW

**Review by edwinnambaje — APPROVED** [review]
LGTM

**Review by Martha-Iradukunda — APPROVED** [review]
LGTM

File: src/services/user.services.js [hunk 1]
```diff
@@ -106,7 +106,6 @@ export default class User {
 
     transporter.sendMail(mailOptions, (err, info) => {
       if (err) return console.error('ERROR SENDING MAIL: ', err);
```
Comment (nshutijonathan) [inline src/services/user.services.js]: remove `console.log` you can simply `return error.message`

**Review by nshutijonathan — CHANGES_REQUESTED** [review]

**Review by Zeden12 — APPROVED** [review]
LGTM

**Review by bencyubahiro77 — APPROVED** [review]

**Review by nshutijonathan — APPROVED** [review]
LGTM

**Review by UmuhireJessie — APPROVED** [review]

**Pull request merged**